packages/http/src/__tests__/fixtures/index.ts

```diff
@@ -274,7 +274,7 @@ export const httpInputs: IHttpRequest[] = [
   {
     method: 'post',
     url: {
-      path: '/',
+      path: '/todos',
```

Comment:
Why this change?

Reply:
Not sure, most probably some leftover. I removed it.